.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = ../../commaai/panda.git
 [submodule "opendbc"]
 	path = opendbc
-	url = ../../commaai/opendbc.git
+	url = git@github.com:belm0/opendbc.git
```

Comment:
temporarily point to pending PR https://github.com/commaai/opendbc/pull/413

Reply:
I can't explain this test failure.    `honda_hrv_touring_2019_can_generated` is the same after my opendbc change, only with a few items shifted around.
```
...................E........................................................CANParser: could not find message 0x0 in DBC honda_hrv_touring_2019_can_generated
python: opendbc/can/parser.cc:140: CANParser::CANParser(int, const std::string &, const std::vector<MessageParseOptions> &, const std::vector<SignalParseOptions> &): Assertion `false' failed.
```

Reply:
it needs to be added to release/files_common

Reply:
thank you, I've added the freed dbc to `files_common`

but I don't think it explains the error regarding the existing honda_hrv_touring_2019_can_generated file

```
CANParser: could not find message 0x0 in DBC honda_hrv_touring_2019_can_generated
```

Reply:
error in HRV test was due to a bug in my carstate refactoring-- fixed